build-kong.sh

```diff
@@ -89,7 +55,18 @@ popd
 cp /kong/COPYRIGHT /tmp/build/usr/local/kong/
 cp /kong/bin/kong /tmp/build/usr/local/bin/kong
 sed -i 's/resty/\/usr\/local\/openresty\/bin\/resty/' /tmp/build/usr/local/bin/kong
+sed -i 's/\/tmp\/build//g' /tmp/build/usr/local/bin/openapi2kong || true
```

Comment:
This isn't a real suggestion because I know there's a pattern here that nobody wants to deviate from, but... it'd be awesome to see these `sed` invocations using a different pattern delimiter so that `/` doesn't need to be escaped. I like `|` or `#`

```suggestion
sed -i 's#/tmp/build##g' /tmp/build/usr/local/bin/openapi2kong || true
```

Reply:
huh yeah in hindsight that's an excellent suggestion. Per the PR description we (Isa and I) are avoiding refactors as much as we can in this series of PR's but for sure. I'd like to lint the entire repository :laughing: